tests/src/tracing/tracecontrol_api/Configuration.cs

```diff
@@ -0,0 +1,110 @@
+using System;
```

Comment:
Can you put all of the tracecontrol_api stuff into a common directory with a common directory, and then perhaps rename the project to "Tracing.Tests.Common"?  This would allow us to add more common stuff in the future, which I'm sure we'll need.